Security/Risk_management/Rapid_Risk_Assessment.mediawiki

```diff
@@ -0,0 +1,356 @@
+| < 10M USD.
+|-
+| <span style="padding: 0 14px; background-color:#E77471;">MAXIMUM</span>
+| > 10M USD.
+|-
+|}
+
+Work-force (productivity-derived) financial costs calculations are included in the RRA spreadsheet.
+
+==== Recording risk impacts ====
+
+
+The risk impact is solely sourced from the RRA when risk is caculated. Thus, the RRA risk impact is the most important
+value being recorded and must be as accurate as possible.
+
+* Tell the team you're now going to look at the impact on confidentiality, availability and integrity of the service.
+** Confidentiality is what happens if the data is leaked (all files zipped up and served on a public site).
+** Availability is what happens if the service is no longer available (DDOS or malfunction, service unreachable).
+** Integrity is what happens if the service data is tampered with (records no longer show trustworthy data, site is
+defaced, etc.)
+* For each, we'll look at how it affects Mozilla's reputation, productivity and finances.
+** Reputation is our public image, both internal and external to Mozilla.
+** Productivity is our ability to work. If a team can't do their regular work, their productivity is affected.
+** Finances represent the cost of an impact. For example, abusing an AWS account and running bitcoin-mining instances
+would cost us money.
+* For each row, there is a summary/reference built-in the template that let you know how the level is set.
+
+As you go through each row, ensure that the team understand which level you're selecting as impact and why.
+
+For example:
+
+Confidentiality=>Reputation HIGH impact would mean that Mozilla would get in tech. news (HN, Ars Technica, etc.)
+if the data was leaked.
+Confidentiality=>Productivity MEDIUM impact would mean that some small Mozilla teams (SG) would be affected for more than
+24h, or a large team (LG) for hours.
+If productivity impact is higher than LOW, it is possible that we occur financial impacts derived from work-force costs.
+
+As you go, ensure that you fill a short rationale on why you selected a level. Remember that anyone reading this
+document will rely on your rationale and need to be able to understand why the level is selected. List affected teams
+and their sizes, for how long for example.
+
+
+==== Recording risk likelihood ====
+
+Our likelihood level recorded during the RRA is very qualitative and of lower accuracy than the impact assessment.
+Ensure that you do not spent too much time arguying about the level recorded - this level will change and the final risk
+calculation takes several data sources in addition to the RRA itself.
+
+Questions to ask:
+
+* Is the service code peer-reviewed?
+* Did the service get any kind of security review, modeling, pen-testing in the past?
+* Did the service suffer any security vulnerability in the past?
+** Did any of the vulnerabilities get exploited or the service compromised?
+* How confident are you that this impact may occur during the next year?
```

Comment:
* Are we aware of ongoing attacks on the service? (brute force attempts, DDoS, ...)